### PR TITLE
added missing index specs

### LINKED ISSUE #692: add n+1 specs on next admin resources
GET /report_realtime_termination_distributions
 GET /report_realtime_origination_performances
 GET /report_realtime_bad_routings
 GET /report_realtime_not_authenticateds

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
closes #692

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -47010,7 +47010,8 @@ ALTER TABLE ONLY sys.sensors
 -- PostgreSQL database dump complete
 --
 
-SET search_path TO gui, public, switch, billing, class4, runtime_stats, sys, logs, data_import;
+SET search_path TO gui, public, switch, billing, class4, grateruntime_stats, sys, logs, data_import
```
Comment (gigorok) [inline db/structure.sql:47013]: undo this changes

**Comment by Ivanov-Anton** [comment]
@senid231 code review, please.
